tsdb/executor.go

```diff
@@ -475,6 +471,12 @@ func (e *SelectExecutor) executeAggregate(out chan *influxql.Row) {
 			}
 		}
 
+		// Perform top/bottom unwraps
+		values, err = e.processTopBottom(values, columnNames)
```

Comment:
This will likely change to `processAggregates` in the future and branch from there, as we are going to add support for `MAX`, `MIN`, `FIRST`, `LAST` to also have additional tags/fields returned.